File: fizzbuzz.py
def fizzbuzz (num) :
    result = []
    for num in range (1, num + 1) :
        if (num % 3 == 0) and (num % 5 == 0):
            result.append ("FizzBuzz")
        elif (num % 3 == 0):
            result.append ("Fizz")
        elif (num % 5 == 0):
            result.append ("Buzz")
        else:
            result.append (str(num))
    return result           

print(fizzbuzz (100))

File: fizzbuzz.ml
let fizzbuzz num =
  let ls = ref [] in
  for i = 1 to num do
    if i mod 3 = 0 && i mod 5 != 0
    then ls := "Fizz" :: !ls
    else if i mod 5 = 0 && i mod 3 != 0
    then ls := "Buzz" :: !ls
    else if i mod 5 = 0 && i mod 3 == 0
    then ls := "FizzBuzz" :: !ls
    else ls := string_of_int (i) :: !ls
  done;
  List.rev (!ls);;

 fizzbuzz 100;;
